### PR TITLE
php-slim: Update to PHP 8 and Slim Framework 4

### PR DESCRIPTION
**Pull request by amotl**
Hi there,

this fixes the list of dependencies, to make the PHP application build and start again. For supporting PHP 8, it was needed to update to Slim Framework 4. Along the lines, as usual, some improvements to the documentation.

With kind regards,
Andreas.

### CODE REVIEW

File: php/index.php [hunk 1]
```diff
@@ -20,91 +27,123 @@ function __construct($config)
...
+ * Insert a post.
  */
-$app->post('/posts', function() use ($app)
+$app->post('/posts', function (Request $req, Response $res, $args = [])
 {
-    $data      = json_decode($app->request->getBody());
-    $user      = $data->user;
-    $text      = $data->text;
-    $image_ref = $data->image_ref;
+    $helper = $this->get('helper');
```
Comment (hammerhead) [inline php/index.php:129]: Indentation here should be the same as for the lines below?

File: php/index.php [hunk 2]
```diff
@@ -119,230 +158,266 @@ function success($status, $result, $contenttype = 'application/json')
...
     $digest  = sha1($content);
-    $ch      = curl_init("{$app->config['blob_url']}guestbook_images/{$digest}");
+    $ch      = curl_init("{$helper->config['blob_url']}guestbook_images/{$digest}");
     curl_setopt($ch, CURLOPT_CUSTOMREQUEST, "PUT");
     curl_setopt($ch, CURLOPT_POSTFIELDS, $content);
     curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
     $result = curl_exec($ch);
     $info   = curl_getinfo($ch);
+    //print("INFO");
+    //print_r($info);
```
Comment (hammerhead) [inline php/index.php:317]: Remove these lines?

**Review by hammerhead — APPROVED** [review]

File: php/documentation.md [hunk 3]
```diff
@@ -45,7 +49,7 @@ $dsn = 'crate:SERVER_IP:4200';
 $conn = new Crate\PDO\PDO($dsn, null, null, null);
 ```
 
-As Crate doesn't support authentication, the other parameters can be left null.
+As CrateDB doesn't support authentication, the other parameters can be left null.
```
Comment (hlcianfagna) [inline php/documentation.md:52]: We actually do support auth now https://crate.io/docs/pdo/en/latest/connect.html#get-a-connection

**Review by hlcianfagna — CHANGES_REQUESTED** [review]

File: php/index.php (hunk 1 above)
Comment (amotl) [reply php/index.php]: Thanks. I've addressed it with 39b07742a.

File: php/index.php (hunk 2 above)
Comment (amotl) [reply php/index.php]: Thanks for spotting. 39b07742a removes the lines.

File: php/documentation.md (hunk 3 above)
Comment (amotl) [reply php/documentation.md]: Thank you. b7454f607 improves the documentation once more, and also addresses this topic.

**Review by hlcianfagna — APPROVED** [review]

**Pull request merged**